frontend/packages/dev-console/src/components/topology/Topology.tsx

```diff
@@ -133,6 +120,16 @@ const Topology: React.FC<TopologyProps & StateProps & DispatchProps> = ({
   onGraphModelChange,
   getStoredGraphModel,
 }) => {
+  const [
+    topologyLayoutData,
+    setTopologyLayoutData,
+    isTopologyLayoutDataLoaded,
+  ] = useUserSettingsCompatibility(
+    TOPOLOGY_LAYOUT_CONFIG_STORAGE_KEY,
+    TOPOLOGY_LAYOUT_LOCAL_STORAGE_KEY,
+    {},
+  );
+  const topologyLayoutDataJson = isTopologyLayoutDataLoaded ? topologyLayoutData : {};
```

Comment:
Same comment about loaded here. I think we _need_ the config loaded before we render this component since we require the data on first render.